src/ddscxx/include/dds/topic/detail/TTopicImpl.hpp

```diff
@@ -187,7 +185,9 @@ dds::topic::detail::Topic<T>::Topic(const dds::domain::DomainParticipant& dp,
 
     this->listener(listener, mask);
 
+#if 0
     this->AnyTopicDelegate::set_copy_out(org::eclipse::cyclonedds::topic::TopicTraits<T>::getCopyOut());
+#endif
```

Comment:
Probably better to just remove these lines?

Reply:
I will look into this with @e-hndrks .

Reply:
Will do.